### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by zello-eugene**
Addresses ANDROID-455
- Target Android 14
- Update the SDK project
- Update sample projects

**_Hide whitespace_** hides some of the unimportant formatting changes.

### CODE REVIEW

File: zello-sdk/src/main/java/com/zello/sdk/Audio.java [hunk 1]
```diff
@@ -179,11 +166,6 @@ private void updateAudioState(@Nullable Intent intent) {
 		String mode = Util.emptyIfNull(intent.getStringExtra(Constants.EXTRA_MODE));
 		if (mode.startsWith(Constants.EXTRA_WA)) {
 			_mode = AudioMode.WEARABLE;
-			try {
-				_wearable = Integer.parseInt(mode.substring(Constants.EXTRA_WA.length()));
-			} catch (NumberFormatException ignore) {
-				_wearable = -1;
-			}
```
Comment (zello-eugene) [inline zello-sdk/src/main/java/com/zello/sdk/Audio.java:186]: Not only wearables are no longer supported, but also the currently selected wearable index was never utilized.

File: zello-sdk-sample-contacts/build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
 	}
 
     dependencies {
```
Comment (adambailey-) [inline zello-sdk-sample-contacts/build.gradle:7]: looks like the indentations are off between this and `repositories` above

**Review by adambailey- — APPROVED** [review]
didn't test but looks fine to me

File: zello-sdk/src/main/java/com/zello/sdk/Sdk.java [hunk 3]
```diff
@@ -849,18 +854,15 @@ public void onReceive(Context context, Intent intent) {
...
 	}
 
 	private void unregisterAppStateReceivers() {
 		if (_context == null) {
 			return;
 		}
-		if (_receiverPackage != null) {
-			_context.unregisterReceiver(_receiverPackage);
-			_receiverPackage = null;
-		}
```
Comment (zello-eugene) [inline zello-sdk/src/main/java/com/zello/sdk/Sdk.java:863]: Package broadcast should not be unregistered here.

File: zello-sdk-sample-contacts/build.gradle (hunk 2 above)
Comment (zello-eugene) [reply zello-sdk-sample-contacts/build.gradle]: Right, AS has gone through so many default formatting styles that it's now all over the place. Re-formatted them all, but that also lit up a lot of changes.

File: zello-sdk/src/main/java/com/zello/sdk/Audio.java [hunk 4]
```diff
@@ -72,18 +73,12 @@ void close() {
...
-			case WEARABLE:
-				return _wearables > 0;
-			default:
-				return false;
-		}
+		return switch (mode) {
+			case SPEAKER -> _sp;
+			case EARPIECE -> _ep;
+			case BLUETOOTH -> _bt;
+			case WEARABLE -> _wearables > 0;
```
Comment (ywZello) [inline zello-sdk/src/main/java/com/zello/sdk/Audio.java:80]: why keep if wearables are not supported anymore?

**Review by nathan-zello — APPROVED** [review]

**Review by jpvasconcellos — APPROVED** [review]

File: zello-sdk/src/main/java/com/zello/sdk/Audio.java (hunk 4 above)
Comment (zello-eugene) [reply zello-sdk/src/main/java/com/zello/sdk/Audio.java:80]: I'm hesitant to just chop it off this time around. I think we need a separate ticket so we can set aside a little bit of time to consider what happens when an older version of the app that reports a wearable connects to the SDK.

**Pull request merged**